docs/optimisation/02-thermal-heuristic.md

```diff
@@ -0,0 +1,82 @@
+# Description of the thermal heuristic
+
+## Introduction
+
+By default, Antares-Simulator performs a linear resolution of the adequacy problem. This involves a special treatment of the integer variables, which are the number of dispatched units (NODU) for each thermal cluster.
+
+The purpose of this section is to describe the heuristic treatment of these integer variables in order to allow the linearisation of the weekly optimisation problem.
+
+Please note that this content is only relevant in case the user chooses a linear resolution of the Antares problem. If the MILP resolution of the adequacy problem is chosen by the user, no heuristic is applied, the problem is solved directly considering integer variables. 
+
+## General presentation of the resolution steps
+
+The linearised resolution of the weekly adequacy problem[^1] is summarized in the diagram below:
+
+![Global diagram](global_diagram.png)
+
+The general principle of the linear resolution is that two iterations of the weekly optimisation problem will be solved:
+- A first resolution in which integer variables are either linearised, or completely removed, depending on the "unit commitment mode" chosen (step 1).
+- A second resolution in which the NODU per cluster is fixed, and only the production level of each thermal cluster is set (step 3).
+
+Between these two steps, a heuristic is used to fix the values of the integer variables which will be used in the second resolution (step 2).
+
+Finally, a final annual heuristic is applied on the NODUs of the whole year to resolve potential inconsistencies caused by the decoupling of weeks.
```

Comment:
Some economic variables are computed after this step (OV COST & OP COST)

Reply:
I added this precision.